spec/sequencescape/api/v2/request_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Sequencescape::Api::V2::Request do
+  subject(:request) { create :library_request_with_poly_metadata }
+
+  describe '#poly_metadata' do
+    # In reality, we will not pass metadatable into the initializer as this factory makes it appear.
+    # Instead we would do this in sequence:
+    # 1. select our metadatable request. e.g.
+    #    r = Sequencescape::Api::V2::Request.find(11180).first
+    # 2. create the new poly metadatum, e.g.
+    #    pm1 = Sequencescape::Api::V2::PolyMetadatum.new(key: 'test_key', value: 'test_value')
+    # 3. set the metadatable on the poly metadatum, e.g.
+    #    pm1.relationships.metadatable = r
+    # 4. save the poly metadatum, e.g.
+    #    pm1.save
+    # If we set the metadatable into the initializer, the save fails as the metadatable is treated as
+    # an attribute and not a relationship.
```

Comment:
Is it worth adding this documentation to the v2 /poly_metadatum file as well, to maximise the chance someone who wants to use it sees how to do it?

Reply:
Done